tests/integration/snowflake/test_snowflake_adapter.py

```diff
@@ -324,3 +324,72 @@ def test_check_count_and_query(sf_adapter):
 
     assert exc.errisinstance(TooManyRecords)
     assert sf_adapter.check_count_and_query.retry.statistics["attempt_number"] == 4
+
+
+def test_generate_schema(sf_adapter):
+    DATABASE, SCHEMA = "SNOWSHU_DEVELOPMENT", "GENERATE_SCHEMA_TEST"
```

Comment:
It' good to provide a short description for every single test case we are adding